packages/rest/src/rest.application.ts

```diff
@@ -264,4 +270,23 @@ export class RestApplication extends Application implements HttpServerLike {
   api(spec: OpenApiSpec): Binding {
     return this.bind(RestBindings.API_SPEC).to(spec);
   }
+
+  /**
+   * Mount an Express router to expose additional REST endpoints handled
+   * via legacy Express-based stack.
+   *
+   * @param basePath Path where to mount the router at, e.g. `/` or `/api`.
+   * @param router The Express router to handle the requests.
+   * @param spec A partial OpenAPI spec describing endpoints provided by the
+   * router. LoopBack will prepend `basePath` to all endpoints automatically.
+   * This argument is optional. You can leave it out if you don't want to
+   * document the routes.
+   */
+  mountExpressRouter(
+    basePath: string,
```

Comment:
Do we allow `RegExp`?

Reply:
An express router can take in `RegExp` so it's possible, but I'll do this in a different PR so this one can be landed and I can also add tests.

Reply:
Maybe it's not practical as OpenAPI spec does not support regexp/wildcard for `path`.